src/app/_utils/extractDescriptionFromContent.ts

```diff
@@ -0,0 +1,13 @@
+export function extractDescriptionFromContent(content: string): string {
+  const lines = content.split('\n')
+  const startIndex = lines[0].startsWith('#') ? 1 : 0
+
+  return (
+    lines
+      .slice(startIndex)
+      .join(' ')
+      .replace(/\s+/g, ' ')
+      .trim()
+      .split('.')[0] + '.'
+  )
+}
```

Comment:
I'm not sure what that function is doing, but I still see `#` characters in the Card preview on my end. Here's the content I used for reference:

```
# Tum edidit somnia pennas

## Non et cogeret adiit de

Lorem markdownum ignis vulnere sede robora e placare figura. Quales per Pheneon
emensas flumina!

## Fata nec indignantia

Dimittite unus considere fugit erat cruor Corythus, pondera parmam cedendo omne
ardent. Mitior cernis donec mane, fata est mentisque oscula relatu, cantat
_numina fallite_, et vere, adyto.

## Favorem geri hunc videre inmansuetumque sua tarda

Venit exit est dextro alta modo illum artesque Minyae Aeaciden procul. Corpora
iura rupem deos sano Caesareo, veteris utque. Deceant cadunt his nulla deos: das
gemitu, corpus abstractus, vidisse?
```

Reply:
Let's discuss this function tomorrow together..